app/views/evaluation_forms/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, "Editing evaluation form" %>
 
-<h1>Editing evaluation form</h1>
+<h1><%= if @evaluation_form.phase.end_date < Date.today then "View" else "Editing" end %> evaluation form</h1>
```

Comment:
reuse helper method instead of duplicating logic
```suggestion
<h1><%= if eval_form_disabled?(@evaluation_form) then "View" else "Editing" end %> evaluation form</h1>
```